src/TropicalGeometry/linear_space.jl

```diff
@@ -365,6 +365,34 @@ function tropical_linear_space(I::MPolyIdeal, nu::Union{Nothing,TropicalSemiring
 end
 
 
+@doc raw"""
+    tropical_linear_space(G::Graph, nu::TropicalSemiringMap; weighted_polyhedral_complex_only::Bool=false)
+
+Return the Bergman fan of the graphic matroid of `G` as a tropical linear space, the tropical semiring map `nu` is used to fix the convention.  If `weighted_polyhedral_complex==true`, will not cache any extra information.
+
+# Examples
+```jldoctest
+julia> G = complete_graph(4)
+Undirected graph with 4 nodes and the following edges:
+(2, 1)(3, 1)(3, 2)(4, 1)(4, 2)(4, 3)
+
+julia> tropical_linear_space(G)
+Min tropical linear space
+
+```
+"""
+function tropical_linear_space(G::Graph, nu::Union{Nothing,TropicalSemiringMap}=nothing; weighted_polyhedral_complex_only::Bool=false)
+    M = zero_matrix(QQ,nv(G),ne(G))
+    for (i,edge) in enumerate(edges(G))
+        M[src(edge),i] = 1
+        M[dst(edge),i] = -1
+    end
```

Comment:
We already have `signed_incidence_matrix`. Could you maybe use that here?

Reply:
`signed_incidence_matrix` only works for directed graphs.  What I have is an undirected graph and I need the signed incidence matrix for a choice of directions (the choice does not matter).

Reply:
That's still something worth putting in its own function for people to use. I don't know if polymake has an acessor for that as well or if this would need your simple loop. Maybe @benlorenz or @lkastner can help out?

Reply:
True, this is actually one of the changes of the Durham workshop that I am preparing right now.

Reply:
Once https://github.com/oscar-system/Oscar.jl/pull/4450 is merged, I am happy to update it.

Reply:
Okay, is fixed in the newest version.